Jenkinsfile

```diff
@@ -5,7 +5,8 @@
 
 properties([
   parameters([
-    choice(name: 'PRODUCT_NAME', choices: 'cmc-citizen-frontend\ncmc-legal-frontend\ndiv-pfe\ndiv-dn\ndiv-rfe\ndiv-da\nprobate-frontend\nprobate-caveats-fe\nccd-api-gateway-web\nccd-case-management-web\nfees-register-api\nfees-register-frontend\nsscs-tribunals-frontend\nsscs-tya-frontend\njui-webapp\nbar-web\nidam-web-admin-idam\nidam-web-public-idam\nsscs-cor-frontend\njui-webapp-web\nccpay-bubble-frontend\nxui-ao-webapp\nxui-mo-webapp\nxui-webapp', description: 'Product + app name, i.e. cmc-citizen-frontend'),
+    string(name: 'DEPRECATION', value: 'WARNING', description: 'The JOB will soon be _DEPRECATED_ once all the frontend applications will be migrated to Azure Kubernetes Service.'),
```

Comment:
this should say what the replacement is, you should be able to use html and a multiline string to make is readable